File: plugins/devkit/devkit-java-tests/testData/inspections/useCoupleFix/UseCoupleOfFactoryMethodInMethodParameterWhenPairCreateUsed_after.java
import com.intellij.openapi.util.Couple;
import com.intellij.openapi.util.Pair;

class UseCoupleOfFactoryMethodInVariableWhenPairCreateUsed {
  void any() {
    takePair(Couple.of("a", "b"));
  }

  <A, B> void takePair(Pair<A, B> pair) {
    // do nothing
  }
}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/useCoupleFix/UseCoupleOfFactoryMethodInMethodParameterWhenPairCreateUsed_after.kt
import com.intellij.openapi.util.Couple
import com.intellij.openapi.util.Pair

class UseCoupleOfFactoryMethodInVariableWhenPairCreateUsed {
  fun any() {
    takePair(Couple.of("a", "b"))
  }

  @Suppress("UNUSED_PARAMETER")
  fun <A, B> takePair(pair: Pair<A, B>?) {
    // do nothing
  }
}
